src/main/webapp/js/studentProfile.js

```diff
@@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
         	$('#statusMessage').css("display", "block")
         	        .attr('class', 'alert alert-danger')
         	        .html('There seems to be a network error, please try again later');
-        	$("html, body").animate({
-        		scrollTop: 0
-        	});
+        	scrollToTop({duration: ''});
```

Comment:
This seems different from one of the other calls where no arguments is passed in, perhaps we should standardize?